### PR TITLE
migrate to github script

### PR DESCRIPTION
**Pull request by lukaselmer**
@sihu @schmijos please review migrate_to_github.rb

### CODE REVIEW

File: lib/renuo/cli/app/migrate_to_github.rb [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    congrats
+  end
+
+  private
+
+  def check_pwd
+    say("Project to transfer is called '#{@project_name}'. Thus, we will create and delete the directories")
+    say("* #{@pwd}/#{@project_name}")
+    say("* #{@pwd}/#{@project_name}.git")
+    agree('Is that ok?')
```
Comment (schmijos) [inline lib/renuo/cli/app/migrate_to_github.rb]: This message is not very clear. What is the command really doing with my local repository?

File: lib/renuo/cli/app/migrate_to_github.rb [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  end
+
+  def transfer_git
+    `git clone --mirror git@git.renuo.ch:renuo/#{@project_name}.git`
+    `cd #{@project_name}.git && hub create -p renuo/#{@project_name}`
```
Comment (schmijos) [inline lib/renuo/cli/app/migrate_to_github.rb]: Maybe we should state the dependencies somewhere. I for example have not installed *hub* on my machine.

File: lib/renuo/cli/app/migrate_to_github.rb [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    `cd #{@project_name}.git && git push --mirror git@github.com:renuo/#{@project_name}.git`
```
Comment (schmijos) [inline lib/renuo/cli/app/migrate_to_github.rb]: This push wont work if there are files in you repo bigger than 100mb (50mb will already give a warning).

File: lib/renuo/cli/app/migrate_to_github.rb [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    `rm -rf #{@project_name}.git`
+  end
+
+  def update_readme
+    agree('Let us update the README.md now. Ready?')
+
+    `git clone git@github.com:renuo/#{@project_name}.git`
+    `cd #{@project_name} && git fetch --all && git checkout develop && git pull && git flow init -d`
+    File.write("#{@project_name}/README.md", File.read("#{@project_name}/README.md").gsub('git.renuo.ch', 'github.com'))
```
Comment (schmijos) [inline lib/renuo/cli/app/migrate_to_github.rb]: nice :+1:

File: lib/renuo/cli/app/migrate_to_github.rb [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+    `cd #{@project_name} && git commit -m 'migrate to github' && git push --set-upstream origin develop`
+    `rm -rf #{@project_name}`
+  end
+
+  def update_readme_loop
+    loop do
+      puts `cd #{@project_name} && git add . && git status && git diff --staged`
+      break if agree('Does this look ok?')
+      ask("Please change it manually in #{@pwd}/#{@project_name}. Hit enter when you are done.")
```
Comment (schmijos) [inline lib/renuo/cli/app/migrate_to_github.rb]: Can I rerun the whole script if I hit `ctrl-c` here?

File: lib/renuo/cli/app/name_display.rb [hunk 6]
```diff
@@ -1,5 +1,17 @@
...
+  end
+
   def display_name(name)
     puts "TODO: display #{name}"
   end
+
+  def delete_current
+    display_name('')
+  end
 end
```
Comment (schmijos) [inline lib/renuo/cli/app/name_display.rb]: Did this app accidentally sneak into this PR?

File: lib/renuo/cli/app/migrate_to_github.rb (hunk 1 above)
Comment (lukaselmer) [inline lib/renuo/cli/app/migrate_to_github.rb]: done - better?

File: lib/renuo/cli/app/migrate_to_github.rb (hunk 2 above)
Comment (lukaselmer) [inline lib/renuo/cli/app/migrate_to_github.rb]: already discussed that with @sihu: it should be installed from here: https://docs.google.com/document/d/1ChUyvSNZfxOmF8oTn8KTax2Cob4cEc1BsNas0WTXCFU/edit# and thus we concluded that it should be installed anyways :hushed: 

but i think you're right. _hub_ is the only special thing, right? then it's done :smiley:

File: lib/renuo/cli/app/migrate_to_github.rb (hunk 3 above)
Comment (lukaselmer) [inline lib/renuo/cli/app/migrate_to_github.rb]: yes, you are right. however, this is not the case for *any* of our repos, so YAGNI, right? :wink:
see https://git.renuo.ch/admin/projects?sort=repository_size_desc

File: lib/renuo/cli/app/migrate_to_github.rb (hunk 4 above)
Comment (lukaselmer) [inline lib/renuo/cli/app/migrate_to_github.rb]: :blush:

File: lib/renuo/cli/app/migrate_to_github.rb (hunk 5 above)
Comment (lukaselmer) [inline lib/renuo/cli/app/migrate_to_github.rb]: nope

File: lib/renuo/cli/app/name_display.rb (hunk 6 above)
Comment (lukaselmer) [inline lib/renuo/cli/app/name_display.rb]: nope, that's a feature :stuck_out_tongue_winking_eye:

File: lib/renuo/cli/app/migrate_to_github.rb (hunk 5 above)
Comment (lukaselmer) [inline lib/renuo/cli/app/migrate_to_github.rb]: in general, the script is _not_ fault tolerant at all...

**Pull request merged**